CMake/Dependencies/libkvsCommonLws-CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include(ExternalProject)
 
 ExternalProject_Add(libkvsCommonLws-download
     GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-    GIT_TAG           3c0ffc9dd26483db3f385cc0b38602a0113e327f
+    GIT_TAG           03dd9ea88bc5474878c7023fecfab63f975f6231
```

Comment:
While you are at it, can you update producer C to this commit? 
270c0b6612cc553d458e316e835471c693f518bd